spec/features/users/user_posts_answer_spec.rb

```diff
@@ -18,4 +18,21 @@
       expect(page).to have_content("Dumb question")
     end
   end
+
+  scenario "they fill out the fomr incorrectly on the question show page" do
```

Comment:
Good idea adding these tests. Without it, we would't have found the bug in the questions form.